go/vt/sqlparser/precedence.go

```diff
@@ -85,8 +85,6 @@ func precedenceFor(in Expr) Precendence {
 			return P4
 		case BangOp:
 			return P3
```

Comment:
Hmm... should we move down the precedences coming before? We're leaving a weird gap in the list of precedences. Probably not important though, all tests are happy